packages/app/src/components/Root/Root.tsx

```diff
@@ -79,6 +80,7 @@ const Root: FC<{}> = ({ children }) => (
       <SidebarSpacer />
       <SidebarDivider />
       <SidebarItem icon={HomeIcon} to="/" text="Home" />
+      <SidebarItem icon={AccountTreeIcon} to="/inventory" text="Inventory" />
```

Comment:
I'm not sure Inventory is the user-facing word we should use. I'm thinking there will be a "Services" view as well as "Websites" and "Libraries".

Reply:
May as well be "My Components" right? Have we settled yet what the nav structure will be in the end? I'm concerned that there could be way too many root entries to be manageable.